core/templates/dev/head/pages/review-test-page/review-test-engine.service.ts

```diff
@@ -20,6 +20,7 @@ oppia.factory('ReviewTestEngineService', [
   function() {
     return {
       getReviewTestQuestionCount: function(numOfSkills) {
+        // numOfSkills should be a non-negative integer.
```

Comment:
I think you should throw an error or return 0 when numOfSkills < 0 as well here

Reply:
Done.